assets/js/modules/analytics/components/dashboard/DashboardGoalsWidget.js

```diff
@@ -55,9 +55,13 @@ function DashboardGoalsWidget() {
 		const profileID = store.getProfileID();
 		const internalWebPropertyID = store.getInternalWebPropertyID();
 
+		const { compareStartDate, compareEndDate, startDate, endDate } = select( CORE_USER ).getDateRangeDates( { offsetDays: DATE_RANGE_OFFSET, compare: true, weekdayAlign: true } );
```

Comment:
Similar to the previous comment by @aaemnnosttv, this would be better readable if it was split between multiple lines. We should only put objects in a single line if they have one or two short properties and the line length overall wouldn't suffer.